Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -567,6 +567,12 @@ void WebProcessPool::createGPUProcessConnection(WebProcessProxy& webProcessProxy
 
     ensureProtectedGPUProcess()->createGPUProcessConnection(webProcessProxy, WTFMove(connectionIdentifier), WTFMove(parameters));
 }
+
+void WebProcessPool::resetGPUProcess()
+{
+    if (auto* process = gpuProcess())
+        process->terminateForTesting();
```

Comment:
Why is this calling terminateForTesting() instead of terminate()? terminateForTesting() calls `gpuProcessExited(ProcessTerminationReason::IdleExit)` which doesn't seem right since this is not an IDLE exit.